crates/holochain/tests/sharded_gossip/mod.rs

```diff
@@ -785,6 +775,51 @@ async fn fullsync_sharded_local_gossip() -> anyhow::Result<()> {
     Ok(())
 }
 
+/*
+
+TODO: rewrite for tx5, or simply learn from it and remove it / rewrite it.
```

Comment:
There is code in the Kitsune tests already to start a bootstrap server. E.g. `start_bootstrap` in `bootstrap_client`. It could be moved to make it easier to get at for other tests but it is there. This can probably be removed